src/thread_task.rs

```diff
@@ -323,7 +323,8 @@ unsafe fn create_filter_sbrow(
         let mut frame = fc.frame_thread_progress.frame.try_write().unwrap();
         frame.clear();
         frame.resize_with(prog_sz, || AtomicU32::new(0));
-        let mut copy_lpf = fc.frame_thread_progress.copy_lpf.try_write().unwrap();
+        // copy_lpf is read during task selection, so we may be blocked here
```

Comment:
Could you add to the comment that it's rare that it actually blocks so it should be fine performance wise, and that potentially the blocking could be removed, but it's not currently worth it?

Reply:
Basically what you have in the PR description.